x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/curation/suggested_documents_callout.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { i18n } from '@kbn/i18n';
+
+import { ENGINE_CURATION_SUGGESTION_PATH } from '../../../routes';
+import { generateEnginePath } from '../../engine';
+
+import { SuggestionsCallout } from '../components/suggestions_callout';
+
+import { CurationLogic } from '.';
+
+export const SuggestedDocumentsCallout: React.FC = () => {
+  const {
+    curation: { suggestion, queries },
+  } = useValues(CurationLogic);
+
+  if (typeof suggestion === 'undefined' || suggestion.status !== 'pending') {
```

Comment:
Just curious why you check for `undefined` rather than falsy.

Reply:
Would be nice to have statuses in an enum or at constants.

Reply:
> Just curious why you check for undefined rather than falsy.

To be honest, I default to checking specifically for undefined so I don't have to remember whether `false`/`0`/`""`/`{}` etc are a valid value for the variable or not.  

> Would be nice to have statuses in an enum or at constants.

So, this field [uses a union]( https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/types.ts#L15) to enforce specific values

```
  status: 'pending' | 'applied' | 'automated' | 'rejected' | 'disabled';
```

So VSCode and tsc _will_ complain if you use a value here that's not one of these strings.  And I can still hover over `stuggestion.status` and it lists all the valid strings.  In fact its kind of more useful than enums because for an `SuggestionStatus` enum it would just say `status: SuggestionStatus`.

When I used enums on Crawler there was some pushback against that and towards using unions.  I dunno which is better tbh.  For what its worth I think using a union still enforces a single "source of truth" for valid values, and I think its unlikely we'd change these valid values (that would be a breaking API change) so I'm not _super_ worried about having to search/replace this.  Anyway `tsc` would let us know which values we forgot to update.

Reply:
OK, maybe adding the strings to a constants file at least. We can deal with that in a different PR if we want.